sdk/python/jobs/automl-standalone-jobs/automl-image-object-detection-task-fridge-items/automl-image-object-detection-task-fridge-items.ipynb

```diff
@@ -1381,7 +1478,10 @@
    "cell_type": "code",
    "execution_count": null,
    "metadata": {
-    "name": "deploy"
+    "name": "deploy",
```

Comment:
why does this one have "name"?

Reply:
object detection ones are used int our docs.. that is why they have names... :)